src/wallet/connect.ts

```diff
@@ -153,10 +153,32 @@ export function supportedSpecs(swo: StarknetWindowObject) {
   return swo.request({ type: 'starknet_supportedSpecs' });
 }
 
-export function onAccountChange(swo: StarknetWindowObject, callback: AccountChangeEventHandler) {
+/**
```

Comment:
These files will be removed as per this PR:
https://github.com/starknet-io/starknet.js/pull/1042